rendering.php

```diff
@@ -0,0 +1,20 @@
+<?php
+//Function looks if page excists and put it in the index.php
+function renderPage() {
+	$page = "index";
+	if (isset($_GET['p'])) {
+		$page = $_GET['p'];
```

Comment:
Oh, en er moet gekeken worden of hier niet een pad in zit. Anders kan ik "C:\eenofanderpad" of "../nogeenpad" doen en allerlei bestanden inzien.